### ISSUE TITLE
nextdns log error

### ISSUE THREAD

**Issue by Fail-Safe** [nextdns/nextdns#38]
Freebsd_amd64 v1.3.3 client on pfSense 2.4.4:
```
[2.4.4-RELEASE][root@pfSense-n1.xxx.yyy]/root: nextdns log
Error: exit status 2
```

Where would the NextDNS service logs be stored by default?

**Comment by rs** [comment]
In system. log but not without changing syslog config. See https://github.com/kardianos/service/issues/198

**Comment by rs** [comment]
Fixed by v1.4.0
